### ISSUE TITLE
IndexOutOfRangeException from Parse method when the input ends with extra spaces

### ISSUE THREAD

**Issue by AlissaSabre** [komutan/NMeCab#13]
`MeCabIpaDicTagger.Parse(string)` throws `IndexOutOfRangeException` if the input string (the `sentence` parameter) contains one or more extra space (U+0020) characters at its end.

I think it should just parse the input, ignoring those extra spaces.  Alternatively, it should indicate the error condition in a more reasonable manner, e.g., throwing `ArgumentException("sentence")`, if such a string should really be considered an error.

Sample code to reproduce the issue (please note a space after れ):
```
using System;
using NMeCab;

namespace NMecabIssue
{
    class Program
    {
        static void Main(string[] args)
        {
            try
            {
                using (var tagger = NMeCabIpaDic.CreateTagger())
                {
                    tagger.Parse("あれ ");
                }
            }
            catch (Exception e)
            {
                Console.Out.Write(e.ToString());
            }
        }
    }
}
```
Observed behaviour: to print an `IndexOutOfRangeException`.
Expected behaviour: to print nothing.
Tested version: LibNMeCab.IpaDicBin v0.10.0-beta (Downloaded with nuget)

**Comment by komutan** [comment]
@AlissaSabre
Thank you good report.
Space characters are specially processed in morphological analysis.
I think, this is a bug of it. I'll investigate and fix it.

**Comment by AlissaSabre** [comment]
Thank you.  BTW, I wrote a small demonstration program [here](https://github.com/AlissaSabre/NMeCabDemo), and I included a workaround on this issue in it; just use System.String.Trim before calling Parse.  (I'm not saying the fix of this issue should be calling Trim.)

**Comment by komutan** [comment]
@AlissaSabre
Thank you for your demonstration program, and your help in improving quality.
I got motivate!

----

I've uploaded there NuGet package that fixed this problem.
- LibNMeCab 0.10.0-beta2
- LibNMeCab.IpaDicBin 0.10.0-bata2

It seems that.. we have to explicitly the newest beta version number with nuget.
Is it strange? https://docs.microsoft.com/ja-jp/nuget/create-packages/prerelease-packages
